articles/how-to-generate-url-and-text-qr-codes-using-node/index.md

```diff
@@ -0,0 +1,335 @@
+
+
+### Uses and importance of QR Codes
+
+Some of the major uses of QR Codes in the world as we see today include:
+
+- In online payment to access account numbers or pay bills
+- Advertisements to land on webpages
+- By companies to advertise and download apps
+- Shopping and Ecommerce
+- Direct clients and potential customers on social media platforms and so much more...
+
+You can refer to additional uses [here](https://www.smartinsights.com/digital-marketing-strategy/8-uses-qr-codes-measurable-marketing-campaign/) or [here](https://www.takeflyte.com/blog/50-creative-uses-of-qr-codes). 
+
+QR Codes are very efficient in sharing data. Multiple devices can access the same data anytime and anywhere without restrictions. One does not have to store the data on their devices and can access them at the needed time, whether once or multiple times.
+
+### Overall Folder Structure
+
+The overall project structure will be as shown below:
+
+> QRcode-Generator (Root Directory)
+>
+> > node_modules (folder)
+> >
+> > views (folder)
+> > > index.ejs (file)
+> > > scan.ejs (file)
+> >
+> > index.js (file)
+
+
+Let's begin by setting up our project directory. Create a folder and name it "QRcode-Generator". Open the folder with Visual Studio Code. Once inside VS Code, open the terminal. You can quickly do this by using the `` Ctrl+` `` shortcut on a PC or `` Control + Shift + ` `` shortcut on a Mac. Create a `package.json` file quickly by running `npm init -y` on the terminal. 
+
+Once created, run the following command:
+
+```bash
+npm i qrcode express body-parser nodemon ejs
+```
+
+This installs the following packages to the project:
+
+- body-parser 
+- ejs (Embedded JavaScript templates)
+- express 
+- qrcode
+- nodemon
+
+Let us add some nodemon configuration in the "package.json" file by adding the "start" and "dev" inside the scripts as follows:
+
+```json
+{
+    "name": "QRCODE-URL-main",
+    "description": "",
+    "version": "1.0.0",
+    "main": "index.js",
+    "scripts": {
+        "start": "npm index.js",
+        "dev": "nodemon index.js",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "body-parser": "^1.19.0",
+        "ejs": "^3.1.6",
+        "express": "^4.17.1",
+        "nodemon": "^2.0.7",
+        "qrcode": "^1.4.4"
+    }
+}
+```

```

Comment:
nodemon is not an absolute necessity. Please make sure you only add packages that are 100% required to keep the tutorial simple and to avoid confusion. 

I think you can skip these steps and ask the reader to start the server using:

```bash
node index.js
```

Reply:
Removed nodemon
Please confirm the changes